extension/chrome/settings/modules/contacts.ts

```diff
@@ -63,7 +63,7 @@ View.run(class ContactsView extends View {
   // --- PRIVATE
 
   private loadAndRenderContactList = async () => {
-    const contacts = await ContactStore.search(undefined, { hasPgp: true, limit: 500, substring: String($('.input-search-contacts').val()) });
+    const contacts = await ContactStore.search(undefined, { hasPgp: true, limit: 500, substring: '' });
```

Comment:
This is wrong - you don't want to pull all of the public keys from storage, and then filter them later (there may be thousands of them). To fix this, you would have to dig into `ContactStore.search` itself, and fix it there (which may be complicated).

Reply:
Oh, I see. Thanks for pointing this out. I'll be closing this PR now.